src/installer/index.ts

```diff
@@ -65,13 +61,15 @@ export default function () {
       console.log('[installer:index.ts] Registered event', event.channel)
     })
     ipcMain.on('installer:checkUpdate', async (_, message) => {
-      new Installer(mainWindow!).checkUpdateOrInstall()
+      console.log(
+        '[installer:index.ts] TODO!! -> implement checkUpdateOrInstall method'
+      )
+      // new Installer(mainWindow!).checkUpdateOrInstall()
```

Comment:
I commented this one out because I could not find its definition, just to be cautious.